server/cmd/mmctl/commands/import.go

```diff
@@ -476,6 +503,10 @@ func importValidateCmdF(command *cobra.Command, args []string) error {
 		return err
 	}
 
+	if maxPostSize == 0 {
+		maxPostSize = model.PostMessageMaxRunesV1
+	}
```

Comment:
nit: I think we should default to V2 probably this would be too strict, apart from that could we determine this value from import file?

Reply:
@isacikgoz Thanks, wanted to discuss this. Could you confirm we'd be safe schema wise on both databases? If so no reasons to keep the v1.

Reply:
Okay, I'll update this case but we may need a more formal review of this if we want to properly deprecate v1 since it's being used in many places still (config, post store, etc. Created https://mattermost.atlassian.net/browse/MM-57842 as follow-up.